src/main/java/edu/ucsb/cs48/s20/demo/entities/Review.java

```diff
@@ -1,35 +1,33 @@
 package edu.ucsb.cs48.s20.demo.entities;
 
-
 import java.util.Objects;
 
-import javax.persistence.Column;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import javax.persistence.JoinColumn;
-import javax.persistence.OneToOne;
+import javax.persistence.*;
 
 @Entity
 public class Review {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
-   
-    @OneToOne
+
+    @ManyToOne
     @JoinColumn(name = "student_id")
     private Student reviewer;
-    
-    @OneToOne
+
+    @ManyToOne
     @JoinColumn(name = "idea_id")
     private ProjectIdea idea;
-    
+
     private Integer rating;
 
-    @Column(columnDefinition = "LONGTEXT")
+    @Column(columnDefinition = "text")
```

Comment:
So unfortunately this solution doesn't work on Heroku for some reason, which is a real shame; we'll need to go back and add error checking code for the length while potentially trying to set the length manually ourselves.